common/src/main/java/com/wynntils/features/user/redirects/ChatRedirectFeature.java

```diff
@@ -587,6 +588,25 @@ protected String getNotification(Matcher matcher) {
         }
     }
 
+    private class ScrollTeleportationSoulPointRedirector extends SimpleRedirector {
```

Comment:
Can't we do this for x soul points? Like when we die, isn't there a 2 soul points lost message?

That would also mean renaming the redirector, adding a capture group to the regex, and adding a new config option

Reply:
![image](https://user-images.githubusercontent.com/34697715/208177315-1ddddfdf-8c53-49ad-b696-33911b678a34.png)

It's a completely different message (actually two messages) with different chatformatting codes. I can do that in another PR.

Reply:
The soul point part is almost the same? You can use * for the color code, and s? to fix the pluralization.

You can leave the you have died part in the chat